wagtail/wagtailadmin/templates/wagtailadmin/notifications/base.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<title>Untitled Document</title>
+<style>
+body {
+	min-width:100%;
+	-webkit-text-size-adjust: none;
+	padding: 0;
+	margin: 0;
+}
+.ReadMsgBody {
+	width: 100%;
+}
+.ExternalClass {
+	width: 100%;
+}
+table {
+	border-collapse:collapse;
+	mso-table-lspace:0pt;
+	mso-table-rspace:0pt;
+}
+table td {
+	border-collapse:collapse;
+}
+td.date a:link {
+	color:#6b6b6b;
+	text-decoration:none;
+}
+@media only screen and (max-width: 568px) {
+body[yahoo] .mobile {
+	width:320px !important;
+}
+body[yahoo] .spacer {
+	width:10px !important;
+}
+body[yahoo] .spacer_height {
+	height:10px !important;
+}
+body[yahoo] .content {
+	width:300px !important;
+}
+body[yahoo] .col-3{
+  float: none !important;
+	width: 100% !important;
+}
+body[yahoo] .force-col {
+	display: block;
+	padding-right: 0 !important;
+	padding-left: 0 !important;
+	padding-bottom: 12px !important;
+}
+body[yahoo] .force-col-last {
+	display: block;
+	padding-right: 0 !important;
+	padding-bottom: 0 !important;
+}
+body[yahoo] .center {
+	text-align:center;
+}
+body[yahoo] .show {
+	display: block !important;
+}
+body[yahoo] .hide {
+	display: none !important;
+}
+body[yahoo] .img {
+	width: 130px !important;
+	float:left;
+}
+body[yahoo] .img img {
+	width:100% !important;
+}
+body[yahoo] .text {
+	width: 150px !important;
+	float:right;
+}
+}
+</style>
+</head>
+{% load i18n %}
+<body style="background-color:#E6E6E6;padding:0px;margin:0px;" yahoo="fix" topmargin="0" bottommargin="0" leftmargin="0" rightmargin="0">
+<table width="600" border="0" align="center" class="mobile" cellspacing="0" cellpadding="0">
+  <tr>
+    <td>
+    	<a href="#"><img width="600" border="0" style="display: block;" alt="" src="{{ settings.BASE_URL }}{% static 'wagtailadmin/images/email-header.jpg' %}" /></a>
```

Comment:
Could this part be made into a block so it's overridable? Some people have rebranded their Wagtail sites.

Docs: http://docs.wagtail.io/en/v1.3.1/advanced_topics/customisation/branding.html
Example: https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailadmin/templates/wagtailadmin/base.html#L8-L10